spec/services/usps_in_person_proofing/enrollment_helper_spec.rb

```diff
@@ -19,11 +19,17 @@
   let(:service_provider) { nil }
```

Comment:
This spec file has gotten pretty loose; I'm planning to submit a follow-up PR that refactors this file. Just an FYI